src/domain/constants/prize.js

```diff
@@ -0,0 +1,17 @@
+export const PRIZE = {
+  FIRST: 'FIRST',
+  SECOND: 'SECOND',
+  THIRD: 'THIRD',
+  FOURTH: 'FOURTH',
+  FIFTH: 'FIFTH',
+  LOSS: 'LOSS',
+};
+
+export const PRIZE_BENEFIT = {
+  FIRST: 2_000_000_000,
+  SECOND: 30_000_000,
+  THIRD: 1_500_000,
+  FOURTH: 50_000,
+  FIFTH: 5_000,
+  LOSS: 0,
+};
```

Comment:
```suggestion
export const PRIZE = {
  FIRST: 'FIRST',
  SECOND: 'SECOND',
  THIRD: 'THIRD',
  FOURTH: 'FOURTH',
  FIFTH: 'FIFTH',
  LOSS: 'LOSS',
};

export const PRIZE_BENEFIT = {
  [PRIZE.FIRST]: 2_000_000_000,
  [PRIZE.SECOND]: 30_000_000,
  [PRIZE.THIRD]: 1_500_000,
  [PRIZE.FOURTH]: 50_000,
  [PRIZE.FIFTH]: 5_000,
  [PRIZE.LOSS]: 0,
};
```

이렇게 해야 제대로 매칭될 것 같네요!?